openverse_catalog/dags/providers/provider_api_scripts/phylopic.py

```diff
@@ -220,11 +221,20 @@ def _get_taxa_details(result):
     return taxa_list, title
 
 
+def _get_filesize(img_url):
+    filesize = requests.head(img_url).headers.get("content-length", "")
+    try:
+        return int(filesize)
+    except ValueError:
+        return None
```

Comment:
Same question here - should we neglect this for now since we have to request each image? I think if we're going to be doing this in other cases, we may want to add a `DelayedRequester::head` function if it doesn't already have one.